src/providers/FetchInterceptorProvider.tsx

```diff
@@ -14,9 +14,14 @@ export function FetchInterceptorProvider() {
       const zapQuestId = sessionStorage.getItem(
         ZAP_QUEST_ID_SESSION_STORAGE_KEY,
       );
+      // zaps flow
       if (request.url.startsWith(envConfig.NEXT_PUBLIC_LIFI_API_URL)) {
         request.headers.append('x-zap-quest-id', zapQuestId || '');
       }
+      // scan page flow
+      if (request.url.startsWith(envConfig.NEXT_PUBLIC_LIFI_API_URL) && request.url.includes('status')) {
```

Comment:
If we're calling `/status` from the zap page, we'll also add this header which might lead to unintended behaviour.

Let's add `const pathname = usePathname();` in the provider and then here add another check
```
if (request.url.startsWith(envConfig.NEXT_PUBLIC_LIFI_API_URL) && request.url.includes('status') && pathname?.includes(AppPaths.Scan)) 
```